File: docs/components_page/components/layout/breakpoints.R
library(dashBootstrapComponents)

row <- div(
  list(
    dbcRow(
      list(
        dbcCol(div(
          "One of three columns"
        ), md = 4),
        dbcCol(div(
          "One of three columns"
        ), md = 4),
        dbcCol(div(
          "One of three columns"
        ), md = 4)
      )
    ),
    dbcRow(
      list(
        dbcCol(div(
          "One of four columns"
        ), width = 6, lg = 3),
        dbcCol(div(
          "One of four columns"
        ), width = 6, lg = 3),
        dbcCol(div(
          "One of four columns"
        ), width = 6, lg = 3),
        dbcCol(div(
          "One of four columns"
        ), width = 6, lg = 3)
      )
    )
  )
)

File: docs/components_page/components/layout/breakpoints.jl
using DashBootstrapComponents

row = html_div([
    dbc_row([
        dbc_col(html_div("One of three columns"), md = 4),
        dbc_col(html_div("One of three columns"), md = 4),
        dbc_col(html_div("One of three columns"), md = 4),
    ]),
    dbc_row([
        dbc_col(html_div("One of four columns"), width = 6, lg = 3),
        dbc_col(html_div("One of four columns"), width = 6, lg = 3),
        dbc_col(html_div("One of four columns"), width = 6, lg = 3),
        dbc_col(html_div("One of four columns"), width = 6, lg = 3),
    ]),
]);
